src/Cafe/HW/Latte/Renderer/Renderer.cpp

```diff
@@ -81,7 +82,7 @@ uint8 Renderer::RGBComponentToSRGB(uint8 cli)
 	return (uint8)(cs * 255.0f);
 }
 
-fs::path _GenerateScreenshotFilename(bool isDRC)
+std::optional<fs::path> _GenerateScreenshotFilename(bool isDRC)
```

Comment:
Since you made all the other functions `static` with no `_` prefix you should update this one as well to match the style. We don't really have a cohesive rule on static vs non-static and either way is fine but I think that functions that belong together should use an uniform style.